### PR TITLE
add express response to res with x-pass-response

### PR DESCRIPTION
**Pull request by p-mcgowan**
closes acr-lfr/generate-it-typescript-server#161 (pending acr-lfr/generate-it-typescript-server#179)

Please ensure your pull request:
- ~Includes additions to the docs, found in the /docs, as required~
- ~The new and changed code you have written is unit tested~ and passes the eslint
- The body of the text for this PR includes enough text for a reviewer to understand the context of the change (see issue)

### CODE REVIEW

File: src/lib/template/helpers/pathParamsToDomainParams.ts [hunk 1]
```diff
@@ -92,17 +92,8 @@ export default function (method: string, pathObject: any, withType: boolean = fa
...
-        'res' + addType(
-        withType,
-        pathObject,
-        undefined,
-        undefined,
-        ));
-    } else {
-      params.push('res' + addType(withType, pathObject));
-    }
+    const type = withType ? ': Response' : '';
```
Comment (j-d-carmichael) [inline src/lib/template/helpers/pathParamsToDomainParams.ts:95]: This is a fairly critical function and will need to close attention to testing - not going to have time on this one till later next week i think

Comment (p-mcgowan) [reply src/lib/template/helpers/pathParamsToDomainParams.ts:95]: sure thing - yeah no rush.
was just a minor refactor as the 2 if else conditions passed the same 2 args to the method, and only the first is actually used so its just a slight streamline

**Comment by p-mcgowan** [comment]
@j-d-carmichael boop. did you get a chance to revisit this one? 

I could do another pass on the params mapper, but based on a few test repos it seemed to behave the same way. Could toss some more tests into it though

**Comment by j-d-carmichael** [comment]
ah sorry @p-mcgowan no i have not, i will make time tonight and get it looked at!

**Comment by p-mcgowan** [comment]
No worries, I also forgot about it but was just revisiting all the old PRs recently

**Comment by j-d-carmichael** [comment]
All good

**Review by j-d-carmichael — APPROVED** [review]

**Pull request merged**